src/hotspot/share/prims/jniCheck.cpp

```diff
@@ -350,24 +350,33 @@ check_is_obj_array(JavaThread* thr, jarray jArray) {
   }
 }
 
+// Arbitrary (but well-known) tag for GetStringChars
+const void* STRING_TAG = (void*)0x47114711;
+
+// Arbitrary (but well-known) tag for GetStringUTFChars
+const void* STRING_UTF_TAG = (void*) 0x48124812;
```

Comment:
Why is this well-known?  This ending in 12 could be an address, do you not want to make this a possible address?